dotcom-rendering/src/components/CardHeadline.tsx

```diff
@@ -35,6 +35,8 @@ type Props = {
 	showQuotes?: boolean; // Even with design !== Comment, a piece can be opinion
 	size?: SmallHeadlineSize;
 	sizeOnMobile?: SmallHeadlineSize;
+	sizeOnTablet?: SmallHeadlineSize;
+	fontGroup?: 'standard' | 'large';
```

Comment:
There is probably a better name for this. Happy to hear suggestions! Also concious that "large" overlaps with the concept of large font...